### PR TITLE
GTM SERVER SIDE Destination

### PR DESCRIPTION
**Pull request by shrouti1507**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

---
This change is [Reviewable](https://reviewable.io/reviews/rudderlabs/rudder-sdk-js/233)

### CODE REVIEW

File: integrations/integration_cname.js [hunk 1]
```diff
@@ -21,6 +21,8 @@ const commonNames = {
   FB_PIXEL: "FACEBOOK_PIXEL",
   "Google Tag Manager": "GOOGLETAGMANAGER",
   GTM: "GTM",
+  "Google Tag Manager Server Side": "GOOGLETAGMANAGERSERVERSIDE",
+  GTM_SERVER_SIDE: "GOOGLETAGMANAGERSERVERSIDE",
```
Comment (utsabc) [inline integrations/integration_cname.js:25]: User can give "GTM SERVER SIDE", "GTM Server Side" u might consider those possibilities?

File: integrations/GoogleTagManagerServerSide/browser.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
+import logger from "../../utils/logUtil";
+
+class GoogleTagManagerServerSide {
+  constructor(config) {
+    this.containerID = config.containerID;
+    this.name = "GOOGLETAGMANAGERSERVERSIDE";
```
Comment (utsabc) [inline integrations/GoogleTagManagerServerSide/browser.js:6]: This name should be GTM_SERVER_SIDE what you used in Config Be

File: integrations/client_server_name.js [hunk 3]
```diff
@@ -10,6 +10,7 @@ const clientToServerNames = {
   CUSTOMERIO: "Customer IO",
   FACEBOOK_PIXEL: "Facebook Pixel",
   GTM: "Google Tag Manager",
+  GTM_SERVER_SIDE: "Google Tag Manager Server Side",
```
Comment (utsabc) [inline integrations/client_server_name.js:13]: here also GTM_SERVER_SIDE

File: integrations/index.js [hunk 4]
```diff
@@ -37,6 +38,7 @@ const integrations = {
   GOOGLEADS: GoogleAds.default,
   VWO: VWO.default,
   GTM: GoogleTagManager.default,
+  GTMSERVERSIDE: GoogleTagManagerServerSide.default,
```
Comment (utsabc) [inline integrations/index.js:41]: use GTM_SERVER_SIDE the one in config be

**Comment by saikumarrs** [comment]
Closing this PR as it was abandoned for many months now.

**Pull request closed without merging**